.changes/unreleased/Features-20240424-145949.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Features
```

Comment:
micro-nit: I'd call this Under the Hood since that's what it is from the perspective of MetricFlow users.

Reply:
Updated.